juriscraper/pacer/download_confirmation_page.py

```diff
@@ -0,0 +1,80 @@
+import re
+from typing import Optional
+
+from ..lib.log_tools import make_default_logger
+from ..lib.string_utils import clean_string
+from .reports import BaseReport
+from .utils import make_docs1_url
+
+logger = make_default_logger()
+
+
+class DownloadConfirmationPage(BaseReport):
+    """An object for querying and parsing appellate PACER documents confirmation
+    download page.
+    """
+
+    def __init__(self, court_id, pacer_session=None):
+        super().__init__(court_id, pacer_session)
+
+    def query(self, pacer_doc_id):
+        """Query the "confirmation download page" endpoint and set the results
+        to self.response.
+
+        :param pacer_doc_id: The internal PACER document ID for the item.
+        :return: a request response object
+        """
+
+        assert (
+            self.session is not None
+        ), "session attribute of DownloadConfirmationPage cannot be None."
+
+        # Make the NDA document URL
+        url = make_docs1_url(self.court_id, pacer_doc_id)
+
+        logger.info("Querying the confirmation page endpoint at URL: %s", url)
+        self.response = self.session.get(url)
+        self.parse()
+
+    @property
+    def data(self):
```

Comment:
It feels like if we're going to have an object for this page, we should gather everything from it that we can. That'd probably include:

 - docket_number
 - cost
 - billable_pages
 - document_description
 - transaction_date

I assume that wouldn't be too hard, and it's probably worth doing, just so things are complete if anybody needs this class for something else. Nothing is more annoying than realizing that a tool is only half done.

Reply:
Great, yeah I was not sure what other data might be useful for this object. Now we get this data.